bench/import_range.go

```diff
@@ -0,0 +1,107 @@
+package bench
+
+import (
+	"context"
+	"fmt"
+	"github.com/pilosa/go-pilosa"
+	"io"
+	"math/rand"
+)
+
+type ImportRange struct {
+	Name         string `json:"name"`
+	MinValue     int64  `json:"min-value"`
+	MinColumnID  int64  `json:"min-column-id"`
+	MaxValue     int64  `json:"max-value"`
+	MaxColumnID  int64  `json:"max-column-id"`
+	Index        string `json:"index"`
+	Frame        string `json:"frame"`
+	Field        string `json:"field"`
+	Iterations   int64  `json:"iterations"`
+	Seed         int64  `json:"seed"`
+	Distribution string `json:"distribution"`
+	BufferSize   uint
+	rng          *rand.Rand
+	HasClient
+}
+
+type ValueIterator struct {
+	actualIterations int64
+	bitnum           int64
+	maxbitnum        int64
+	minvalue         int64
+	maxvalue         int64
+	mincol           int64
+	maxcol           int64
+	avgdelta         float64
+	lambda           float64
+	rng              *rand.Rand
+	fdelta           func(z *ValueIterator) float64
+}
+
+// Init generates import range data based on
```

Comment:
This comment is incomplete. Also, comments should all end with a period (see others in this PR).